CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Version changelog
 
+## [Release] Release v0.232.0
+
+New features for Databricks Asset Bundles:
+This release adds support for managing AI/BI dashboards as part of your bundle configuration. The `bundle generate` command is updated to support producing dashboard bundle configuration as well as dashboard payloads.
```

Comment:
`dashboard payloads` is a bit vague, should we clarify this to say `a serialized JSON representation of the dashboard`?